frontend/src/styles/_uswds-theme.scss

```diff
@@ -18,6 +18,27 @@ in the form $setting: value,
   // Ensure utility classes always override other styles
   $utilities-use-important: true,
 
+  $theme-utility-breakpoints: (
+    "card": false,
+    // 160px
+    "card-lg": false,
+    // 240px
+    "mobile": false,
+    // 320px
+    "mobile-lg": true,
+    // 480px
+    "tablet": true,
+    // 640px
+    "tablet-lg": true,
+    // 880px
+    "desktop": true,
+    // 1024px
+    "desktop-lg": false,
+    // 1200px
+    "widescreen": false,
+    // 1400px
+  ),
+
```

Comment:
Is the formatter moving these inline comments to their own line? I find this much easier to read: 

<img width="438" alt="image" src="https://github.com/HHS/grants-equity/assets/409279/ba6d56af-847f-41b2-ab61-1cd22a64729b">

Reply:
Yeah, that was the formatter.

Reply:
Might be a setting that allows inline CSS comments. 🤔

Reply:
What is this doing? Are we overriding the defaults?

Reply:
By default only `mobile-lg`, `tablet`, and `desktop` breakpoints are enabled. This PR overrides the breakpoint map to also enable `tablet-lg`.

Reply:
…so that we can have all 4 PDF links beside each other on screens slightly smaller than `desktop`. Otherwise the link images are HUGE.

Reply:
Seems like prettier is super opinionated on not having comments on the same line as code, so much so that there is no option to disable it for a file, and for some reason, `// prettier-ignore` ir `/* prettier-ignore */` isn't working on this block of scss code: https://github.com/prettier/prettier/issues/807

Reply:
Maybe we don't need the inline comments. 

```css
  /* Override default breakpoints, 
     see https://designsystem.digital.gov/utilities/layout-grid/#utilities-settings */ 
  $theme-utility-breakpoints: (
    "card": false,
    "card-lg": false,
    "mobile": false,
    "mobile-lg": true,
    "tablet": true,
    "tablet-lg": true,
    "desktop": true,
    "desktop-lg": false,
    "widescreen": false,
   ),
```